doc/source/dask-on-ray.rst

```diff
@@ -9,8 +9,8 @@ entire Dask ecosystem can be executed on top of Ray.
 
 .. note::
 
-  Note that Ray does not currently support object spilling, and hence cannot
-  process datasets larger than cluster memory. This is a planned feature.
+  Note that Ray does not yet support object spilling, and hence cannot
+  process datasets larger than cluster memory. This feature is currently being developed.
```

Comment:
But object spilling won't be turned on by default until the 1.3 release, right? Should we wait to call out that feature until it's on by default in the most recent release, or should we call it out and include instructions for turning it on?

Reply:
I think it is fine because we already have a doc in 1.2.0 about how to turn on the object spilling. The feature is probably not as stable as 1.3.0, but we can mention the feature is under active development and report issues if they face any.